ci-operator/step-registry/openstack/deprovision/bastionproxy/openstack-deprovision-bastionproxy-commands.sh

```diff
@@ -11,6 +11,30 @@ fi
 
 export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml
 CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
+BASTION_FIP=$(<"${SHARED_DIR}"/BASTION_FIP)
+
+# configure the local container environment to have the correct SSH configuration
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${BASTION_USER}:x:$(id -u):0:${BASTION_USER:-centos} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    fi
+fi
+# shellcheck disable=SC2140
+SSH_ARGS="-o ConnectTimeout=10 -o "StrictHostKeyChecking=no" -i ${CLUSTER_PROFILE_DIR}/ssh-privatekey"
+SSH_CMD="ssh ${SSH_ARGS} ${BASTION_USER}@${BASTION_FIP}"
+SCP_CMD="scp ${SSH_ARGS}"
+
+>&2 echo "Collecting squid logs from 'bastionproxy-$CLUSTER_NAME'"
+$SSH_CMD bash - <<EOF
```

Comment:
This causes CI to fail with `ssh: Could not resolve hostname : Name or service not known`.

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/20672/rehearse-20672-pull-ci-openshift-installer-release-4.9-e2e-openstack-byon/1420630624082857984/artifacts/e2e-openstack-byon/openstack-deprovision-bastionproxy/build-log.txt

That's because `${SHARED_DIR}/BASTION_FIP` is empty, see below.